src/Agent.Service/Windows/Program.cs

```diff
@@ -14,6 +10,12 @@ static class Program
         /// </summary>
         static void Main(String[] args)
         {
+            if (args != null && args.Length == 2 && args[0].Equals("init", StringComparison.InvariantCultureIgnoreCase))
+            {
+                string source = "VstsAgentService";
```

Comment:
Does this always match the service name? What if multiple agent Windows services are installed on the same box? Or is support for that that not implemented yet?